.github/workflows/ci.yml

```
@@ -1,32 +1,53 @@
 ## This is the name of the CI
 name: CI
 
-## These are the actions that trigger the CI to run
-on: [push, pull_request]
+## These are the triggers for the CI to run
+on: [ push , pull_request ]
 
 jobs:
+
   build:
-    # TODO: Eventually include different OS's in the matrix
+    strategy:
+      fail-fast: false
+      matrix:
+        # We build on our supported version of coq and the master version
+        coq_version: [ '8.13' , 'latest' ]
+        ocaml_version: [ '4.11-flambda' ]
```

I think this will work?
```suggestion
        coq_version: [ '8.13' , 'latest' ]
        ocaml_version: [ '4.11-flambda' ]
        include:
        - coq_version: '8.13'
          extra_gh_reportify: '--warnings'
```